docs/_articles/en/v3.x/release-information.md

```diff
@@ -12,6 +12,25 @@ We publish the latest Code Analyzer monthly.
 * Run `sfdx plugins:update` and `sfdx plugins:update --help` to update Code Analyzer and help to the latest version.
 * Follow these [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in) to update Code Analyzer
 
+## [v3.9.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.9.0) (02-07-2023)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.9.0...v3.8.0)
+
+### Release Summary
+
+* NEW (GraphEngine): We added a new graph-based rule, `UnimplementedTypeRule`. This rule identifies abstract classes and interfaces that are non-global and are missing implementations or extensions. See [UnimplementedTypeRule](./en/v3.x/salesforce-graph-engine/rules/#UnimplementedTypeRule) for more information.
```

Comment:
The link needs to use lowercase `#unimplementedtyperule` instead of camel-cased.